tests/py3_driver_testers.txt

```diff
@@ -23,6 +23,8 @@ AnyTone_778UV_VOX,+Retevis_RT95,13-Nov-2022
 AnyTone_5888UV,@kk7ds,9-Dec-2022
 AnyTone_OBLTR-8R,@KC9HI,17-Dec-2022
 AnyTone_TERMN-8R,@KC9HI,17-Dec-2022
+Baojie_BJ-218,@AA0RD,28-Dec-2022
```

Comment:
The @ thing needs to reference a github ID. So you should put @Old-Phart here so that the generated page links properly.